DAC/src/ringbuffer.c

```diff
@@ -0,0 +1,65 @@
+/*
+ * ringbuffer.c
+ *
+ *  Created on: Feb 6, 2018
+ *      Author: sharana
+ *      Description: Program to initiate a ring buffer.
+ */
+#include"ringbuffer.h"
```

Comment:
It is better convention to have a space between the #include directive and the actual filename, in my opinion.